index.html

```diff
@@ -0,0 +1,211 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/085384e239.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="style.css">
+    <title>My Portfolio</title>
+</head>
+
+<body>
+    <!----------navbar setion------------------->
+    <nav class="nav-container">
+        <div class="sidebar-right" id="sidebarright">
+            <div class="cross"><a href=""><i class="fa-sharp fa-solid fa-circle-xmark"></i></a></div>
+            <div><a href="">Portfolio</a></div>
+            <div><a href="">About</a></div>
+            <div><a href="">Contact</a></div>
+
+        </div>
+        <h1 class="item-logo">Hyder</h1>
+        <ul class="items-menu">
+            <a href="#">
+                <li>Portfolio</li>
+            </a>
+            <a href="#">
+                <li>About</li>
+            </a>
+            <a href="#">
+                <li>Contact</li>
+            </a>
+            <a href="#">
+                <li><i class="fa-solid fa-envelope"></i></li>
+            </a>
+
+        </ul>
+        <a href="#">
+            <li><i class="fa-solid fa-bars" onclick="openSideBar()"></i></li>
+        </a>
+
+    </nav>
+    <!--------------Hero section------------------------------------------->
+    <div class="hero-section">
+        <div class="hero-row">
+            <div class="hero-left">
+                <a href="#"><i class="fa-brands fa-github"></i></a>
+                <a href="#"><i class="fa-brands fa-linkedin"></i></a>
+                <a href="#"><i class="fa-brands fa-twitter"></i></a>
+                <a href="#"><i class="fa-solid fa-m"></i></a>
+            </div>
+            <div class="hero-right">
+                <h1>Hi There.<br>I am Hyder</h1>
+                <h5>I'm a web Developer</h5>
+                <p>I can help you build a product , feature or website. Look through some of my work and experience! If
+                    you like what you
+                    see and have a project you need coded,
+                    don’t hesitate to contact me.</p>
+            </div>
+            <div class="hero-right1">
+                <img src="Illustration-header.png">
+            </div>
+        </div>
```

Comment:
- [ ] while you have made a great effort to accomplish this, it seems like you still have a bit of check to make to ensure that your design matches the Figma design.
- [ ] I have included a screenshot below :point_down: that will make you notice the differences even clearly.

|Your Design| Figma Design|
|:--------------:|:-----------------:|
|![image](https://user-images.githubusercontent.com/99715640/212488990-1ee5ae3f-9932-44e1-b25a-c6139cf7c3cc.png)|![image](https://user-images.githubusercontent.com/99715640/212489011-f6185a7c-a3f8-4af8-8cf2-e06e70292e9e.png)|

- [ ] I think you might have to download some fonts like `Poppins`, `Inter` , and `Crete Round` from [Google](fonts.google.com) so that you can use them as per design.
- [ ] Please check the `font-size` of your elements to be sure that they match design.
- [ ] As you have seen :point_up: , our hamburger menu button is snot showing up, and I think you should for possible spelling error or wrong file paths.
- [ ] Please try to implement these changes according to figma design.
- [ ] [OPTIONAL] I suggest that you focus on building the mobile version of your portfolio first before struggling with desktop version. This will save your time a lot